frontend/src/v5/ui/routes/viewer/tickets/ticketsForm/ticketGroups/ticketGroups.component.tsx

```diff
@@ -100,7 +100,9 @@ export const TicketGroups = ({ value, onChange, onBlur }: TicketGroupsProps) =>
 	};
 
 	// If there is no  group it gets a key to get identified within the groups array
-	const onSetEditGroup = (type) => (index) => setEditingOverride({ override: cloneDeep(settingsFormGroups?.[index]) || { key: count++ }, type });
+	const onSetEditGroup = (type) => (index) => {
+		setEditingOverride({ override: cloneDeep(state?.[type]?.[index]), type, editing:true });
```

Comment:
This is not a change request, but in the future, could we rename "editingOverride" to something else, like sidePanelOverride? It is confusing to have a flag property "editing" for something called "editingOverride"